AmplifyPlugins/Predictions/AWSPredictionsPlugin/Dependency/AWSComprehendAdapter.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSComprehend
+
+class AWSComprehendAdapter: AWSComprehendBehavior {
+    let awsComprehend: ComprehendClient
```

Comment:
Maybe use `ComprehendClientProtocol` instead? Or is the actual implementation needed for some reason?

Reply:
TIL about the ClientProtocols. Thanks for pointing that out; that allows us to remove a lot of unnecessary code 🦾 
Addressed in [0e8078c](https://github.com/aws-amplify/amplify-swift/pull/2895/commits/0e8078c01c74b6070d7249a96baecf69690e19d0)